Marlin/Configuration_adv.h

```diff
@@ -804,4 +804,114 @@
 //#define EXPERIMENTAL_I2CBUS
 #define I2C_SLAVE_ADDRESS  0 // Set a value from 8 to 127 to act as a slave
 
+
+
+
+
```

Comment:
As special as this feature is, there's no need for quite so many blank lines here. Remember, whatever is added here needs to be added to all other `Configuration_adv.h` files.